### PR TITLE
Re-order build tasks to publish to ballerina central

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

### CODE REVIEW

**Review by ThisaruGuruge — APPROVED** [review]

**Review by aashikam — APPROVED** [review]
LGTM

File: java.jdbc-ballerina/build.gradle [hunk 1]
```diff
@@ -247,6 +241,8 @@ task ballerinaPublish {
                     commandLine 'sh', '-c', "$distributionBinPath/bal push"
                 }
             }
+        } else {
+            println("Central Access Token is Not Present")
```
Comment (ThisaruGuruge) [inline java.jdbc-ballerina/build.gradle:245]:
```suggestion
            throw new InvalidUserDataException("Central Access Token is Not Present")
```

File: java.jdbc-ballerina/build.gradle [hunk 2]
```diff
@@ -226,17 +230,7 @@ task ballerinaBuild {
...
-        }
-
-        if (project.version.endsWith('-SNAPSHOT')) {
-            throw new GradleException("Stdlib not pushed to central for snapshot version '${project.version}'.")
-        } else {
-            if (ballerinaCentralAccessToken == null) {
-                throw new GradleException("Central push failed invalid access token.")
-            }
-
+        if (ballerinaCentralAccessToken != null) {
```
Comment (ThisaruGuruge) [inline java.jdbc-ballerina/build.gradle:233]:
```suggestion
        if (project.version.endsWith('-SNAPSHOT')) {
            return
        }
        if (ballerinaCentralAccessToken != null) {
```

File: java.jdbc-ballerina/build.gradle [hunk 3]
```diff
@@ -247,6 +244,8 @@ task ballerinaPublish {
                     commandLine 'sh', '-c', "$distributionBinPath/bal push"
                 }
             }
+        } else {
+            throw new InvalidUserDataException("Central Access Token is Not Present")
```
Comment (ThisaruGuruge) [inline java.jdbc-ballerina/build.gradle:248]: Shall we add the condition to check the `SNAPSHOT` in the version?
(Ref: https://github.com/ballerina-platform/module-ballerina-encoding/pull/80)

Comment (niveathika) [reply java.jdbc-ballerina/build.gradle:248]: It has been added in d8d4d89

**Pull request merged**